examples/nodejs/load-gen/request-coalescing.ts

```diff
@@ -199,8 +199,11 @@ const loadGeneratorOptions: BasicLoadGenOptions = {
    * but it will also increase CPU consumption.  As CPU usage increases and there
    * is more contention between the concurrent function calls, client-side latencies
    * may increase.
+   *
+   * **Note**: You are likely to see degraded performance if you increase this above 50
+   * and observe elevated client-side latencies.
    */
-  numberOfConcurrentRequests: 200,
```

Comment:
This should have never been 200 and leads our customers into seeing poor client-side latencies if they ever use this